scripts/console.ts

```diff
@@ -283,29 +303,43 @@ export async function createClones(
     return [tokenAddr, governorAddr];
 }
 
-// --- Deployment and input handling helpers ---
+// --- Deployment helpers ---
 
-// source of most of these are from Radicle:
-// https://github.com/radicle-dev/radicle-contracts/blob/7070d51fdd8f99790b8fb4e4c953351fd417839a/src/deploy-to-network.ts
+const JSON_NUM_SPACES = 4;
 
-async function deploy<T extends Contract>(name: string, fn: () => Promise<T>): Promise<T> {
+// eslint-disable-next-line @typescript-eslint/no-var-requires, node/no-unpublished-require
+let deployments: Deployments = require(`../deployments.json`);
+
+async function trackDeployment<T extends Contract>(
```

Comment:
I like that we're using one function to deploy GitConsensus, TokenFactory, and GovernorFactory.

Reply:
Yup, this is a super useful function.